Source/Editor/Cooker/Steps/DeployDataStep.cpp

```diff
@@ -214,6 +214,30 @@ bool DeployDataStep::Perform(CookingData& data)
                 FileSystem::NormalizePath(srcDotnet);
                 LOG(Info, "Using .NET Runtime {} at {}", TEXT("Host"), srcDotnet);
 
+                // Get major Version
+                Array<String> pathParts;
+                srcDotnet.Split('/', pathParts);
+                String version;
+                for (int i = 0; i < pathParts.Count(); i++)
+                {
+                    if (pathParts[i] == TEXT("runtimes"))
+                    {
+                        Array<String> versionParts;
+                        pathParts[i - 1].Split('.', versionParts);
+                        const String majorVersion = versionParts[0].TrimTrailing();
```

Comment:
This could check if array is non-empty to prevent errors in case dotnet changes folders structure.